src/OgDeleteOrphansBase.php

```diff
@@ -128,6 +128,9 @@ protected function query(EntityInterface $entity) {
    */
   protected function deleteOrphan($entity_type, $entity_id) {
     $entity = $this->entityTypeManager->getStorage($entity_type)->load($entity_id);
+    if (!$entity) {
```

Comment:
Some implementations delete og related in their own delete functions (e.g. ogmenu). The queue implementation should not fail for these entities.